kernel/kernel-api/src/main/java/io/delta/kernel/expressions/Literal.java

```diff
@@ -157,17 +171,36 @@ private Literal(Object value, DataType dataType) {
         this.dataType = dataType;
     }
 
-    public Object value() {
-        return value;
-    }
-
-    @Override
-    public Object eval(Row record) {
+    /**
+     * Get the literal value. If the value is null a {@code null} is returned. Otherwise one
+     * of the following types based on the literal data type:
+     *
+     * <u>
+     * <li>BOOLEAN -> {@link Boolean}</li>>
```

Comment:
what do these link to?

Reply:
They link to the specific class referred with `@link`